### ISSUE TITLE
dump_all() got an unexpected keyword argument 'sort_keys' when save .nemo file

### ISSUE THREAD

**Issue by MelikaBahmanabadi** [NVIDIA/NeMo#2674]
Hi,
I fine-tuned the QuartzNet15x5Base-En model with 235 hours dataset of Persian Mozilla Common Voice 3 months ago
 and saved the check point file. Now I resorted the CKPT file and start fine-tuning it with another Persian dataset. When I want to save the .nemo file after fine-tuning it has error **dump_all() got an unexpected keyword argument 'sort_keys'** . How caan I solve this error? 
Thanks

**Environment overview**

Environment location: Google Colab
Method of NeMo install: 

BRANCH = 'r1.0.0rc1'
!python -m pip install git+https://github.com/NVIDIA/NeMo.git@$BRANCH#egg=nemo_toolkit[all]

Learning Rate: 0.002

**Environment details**

OS version : "Ubuntu20.04.3 LTS"
PyTorch version : "1.7.1"

**Comment by titu1994** [comment]
Can you remove the rc1 branch and the r1.2.0?

**Comment by MelikaBahmanabadi** [comment]
> Can you remove the rc1 branch and then try r1.2.0?

Thanks. It works well.
